logstash-rs/src/error.rs

```diff
@@ -0,0 +1,19 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum Error {
+    #[error("failed to send {0} to channel")]
```

Comment:
Almost every time it would be `Sender/Receiver half dropped`, correct?

Reply:
Yes